source/administration/bulk-export-data.rst

```diff
@@ -0,0 +1,320 @@
+-----------
+.. raw:: html
+
+  <table width="100%" border="1" cellpadding="5px" style="margin-bottom:20px;">
+    <tr class="row-odd">
+      <th class="head">Field name</th>
+      <th class="head">Type</th>
+      <th class="head">Description</th>
+    </tr>
+    <tr class="row-odd">
+      <td valign="middle">username</td>
+      <td valign="middle">string</td>
+      <td>The user’s username. This is the unique identifier for the user.</td>
+    </tr>
+    <tr class="row-odd">
+      <td valign="middle">email</td>
+      <td valign="middle">string</td>
+      <td>The user’s email address.</td>
+    </tr>
+    <tr class="row-odd">
+      <td valign="middle">auth_service</td>
+      <td valign="middle">string</td>
+      <td>The authentication service used for this user account. <br>
+        <kbd>""</kbd> or not provided - password authentication.<br>
+        <kbd>"gitlab"</kbd> - GitLab authentication.<br>
+        <kbd>"ldap"</kbd> - LDAP authentication (E10 and E20)<br>
+        <kbd>"saml"</kbd> - Generic SAML based authentication (E20)<br>
+        <kbd>"google"</kbd> - Google OAuth authentication (E20)<br>
+        <kbd>"office365"</kbd> - Microsoft Office 365 OAuth Authentication (E20)</td>
+    </tr>
+    <tr class="row-odd">
+      <td valign="middle">auth_data</td>
+      <td valign="middle">string</td>
+      <td>The authentication data if <kbd>auth_service</kbd> is used. The value depends on the <kbd>auth_service</kbd> that is specified.<br>
+        The data comes from the following fields for the respective auth_services:<br>
+        <kbd>""</kbd> or not provided - must be omitted.<br>
+        <kbd>"gitlab"</kbd> - The value of the Id attribute provided in the Gitlab auth data.<br>
+        <kbd>"ldap"</kbd> - The value of the LDAP attribute specified as the "ID Attribute" in the Mattermost LDAP configuration.<br>
+        <kbd>"saml"</kbd> - The value of the SAML Email address attribute.<br>
+        <kbd>"google"</kbd> - The value of the OAuth Id attribute.<br>
+        <kbd>"office365"</kbd> - The value of the OAuth Id attribute.</td>
+    </tr>
+    <tr class="row-odd">
+      <td valign="middle">nickname</td>
+      <td valign="middle">string</td>
+      <td>The user’s nickname.</td>
+    </tr>
+    <tr class="row-odd">
+      <td valign="middle">first_name</td>
+      <td valign="middle">string</td>
+      <td>The user’s first name.</td>
+    </tr>
+    <tr class="row-odd">
+      <td valign="middle">last_name</td>
+      <td valign="middle">string</td>
+      <td>The user’s last name.</td>
+    </tr>
+    <tr class="row-odd">
+      <td valign="middle">position</td>
+      <td valign="middle">string</td>
+      <td>The user’s position.</td>
+    </tr>
+    <tr class="row-odd">
+      <td valign="middle">roles</td>
+      <td valign="middle">string</td>
+      <td>The user’s roles. Will be one of the following values:<br>
+        <kbd>"system_user"</kbd><br>
+        <kbd>"system_admin system_user"</kbd></td>
```

Comment:
Remove these 2 lines. I don't think they need to be specified here and will only go out of date and confuse people.